DESCRIPTION

```diff
@@ -52,7 +52,7 @@ Suggests:
 Remotes: hadley/scales,
     hadley/svglite,
     jimhester/withr,
-    tidyverse/rlang
+    lionel-/rlang@1def2e2cffaaa
```

Comment:
I assume there will be an rlang PR soon?

Reply:
There is already one: https://github.com/r-lib/rlang/pull/471